web/locales/de.json

```diff
@@ -27,6 +27,7 @@
         "back": "zurück",
         "source": "Quelle",
         "fossilfuels": "fossile Brennstoffe",
+        "carbonintensity": "CO2 Konzentration",
```

Comment:
How do we do? Do you approve this PR and do you changes in a separate PR, or do you want to make any changes like **ThierryOllivero** did in 125647c?